src/node/desktop/src/main/preferences/file-preferences.ts

```diff
@@ -34,7 +34,11 @@ class FilePreferences extends DesktopOptions {
     const desktopIni = userConfigDir.completePath(INI_FILE).getAbsolutePath().replace('rstudio', 'RStudio');
 
     if (FilePath.existsSync(desktopIni)) {
-      this.properties = PropertiesReader(desktopIni);
+      try {
+        this.properties = PropertiesReader(desktopIni);
+      } catch (err: unknown) {
+        console.error(err.message); // too early in startup for logging
```

Comment:
Maybe we could add some additional text to let the user know we couldn't migrate their preferences?

Just in case they open RStudio and notice their preferences are different/missing, if they see the console error message, they'll also see the reason for the preference change in addition to the error message.